Sources/BraveWallet/Crypto/Onboarding/RestoreWalletView.swift

```diff
@@ -188,26 +194,30 @@ private struct RestoreWalletView: View {
     .onChange(of: recoveryWords) { [recoveryWords] newValue in
       handleRecoveryWordsChanged(oldValue: recoveryWords, newValue: newValue)
     }
+    .navigationBarBackButtonHidden(isShowingCreatingWallet)
+    .overlay {
+      if isShowingCreatingWallet {
+        CreatingWalletView()
+      }
+    }
```

Comment:
.overlay seems like not going to make content in fullscreen
![Simulator Screenshot - iPhone 15 Pro - 2023-12-06 at 11 18 15](https://github.com/brave/brave-ios/assets/1187676/94dcc4ae-de7e-4bca-a457-014ccfc3070b)